lib/mix/test/mix/tasks/deps_test.exs

```diff
@@ -1,260 +1,355 @@
-Code.require_file "../../test_helper.exs", __DIR__
+Code.require_file("../../test_helper.exs", __DIR__)
 
 defmodule Mix.Tasks.DepsTest do
   use MixTest.Case
 
   defmodule DepsApp do
     def project do
-      [app: :deps, version: "0.1.0",
-       deps: [
-         {:ok, "0.1.0",         github: "elixir-lang/ok"},
-         {:invalidvsn, "0.2.0", path: "deps/invalidvsn"},
-         {:invalidapp, "0.1.0", path: "deps/invalidapp"},
-         {:noappfile, "0.1.0",  path: "deps/noappfile"},
-         {:nosemver, "~> 0.1",  path: "deps/nosemver"},
-       ]]
+      [
+        app: :deps,
+        version: "0.1.0",
+        deps: [
+          {:ok, "0.1.0", github: "elixir-lang/ok"},
+          {:invalidvsn, "0.2.0", path: "deps/invalidvsn"},
+          {:invalidapp, "0.1.0", path: "deps/invalidapp"},
+          {:noappfile, "0.1.0", path: "deps/noappfile"},
+          {:nosemver, "~> 0.1", path: "deps/nosemver"}
+        ]
+      ]
     end
   end
 
   defmodule SuccessfulDepsApp do
     def project do
-      [app: :sample, version: "0.1.0",
-       deps: [
-         {:ok, "0.1.0", path: "deps/ok"}
-       ]]
+      [
+        app: :sample,
+        version: "0.1.0",
+        deps: [
+          {:ok, "0.1.0", path: "deps/ok"}
+        ]
+      ]
     end
   end
 
   defmodule ReqDepsApp do
     def project do
-      [app: :req_deps, version: "0.1.0",
-       deps: [
-         {:ok, ">= 2.0.0",  path: "deps/ok"},
-         {:noappfile,       path: "deps/noappfile", app: false},
-         {:apppath,         path: "deps/noappfile", app: "../deps/ok/ebin/ok.app"}
-       ]]
+      [
+        app: :req_deps,
+        version: "0.1.0",
+        deps: [
+          {:ok, ">= 2.0.0", path: "deps/ok"},
+          {:noappfile, path: "deps/noappfile", app: false},
+          {:apppath, path: "deps/noappfile", app: "../deps/ok/ebin/ok.app"}
+        ]
+      ]
     end
   end
 
   ## deps
 
   test "prints list of dependencies and their status" do
-    Mix.Project.push DepsApp
+    Mix.Project.push(DepsApp)
 
-    in_fixture "deps_status", fn ->
-      Mix.Tasks.Deps.run []
+    in_fixture("deps_status", fn ->
```

Comment:
Please re-run the formatter on latest master as it won't add parens for `in_fixture/2`.